content/en/docs/reference/helper-libraries/dra-driver-controller.md

```diff
@@ -0,0 +1,112 @@
+---
+title: DRA Resource Driver controller helper library
+description: Handles most of functionality common for all DRA resource drivers' controllers.
+weight: 10
+---
+
+<!-- overview -->
+
+When implementing
+[Dynamic Resource Allocation](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) (DRA)
+resource driver controller, the communication with the scheduler is done through API objects.
+
+This library implements some of the functionality that is common for all resource
+drivers, regardless of the actual resource managed or allocation logic. This includes, but isn't restricted to:
+
+- watching and caching of {{< glossary_tooltip term_id="resource-claim" text="ResourceClaims">}}
+- updating the ResourceClaim when the allocation or deallocation is done
+- watching and caching of {{< glossary_tooltip term_id="pod-scheduling-context" text="PodSchedulingContext">}}
+- updating the PodSchedulingContext
+- registering with Kubernetes cluster to handle particular {{< glossary_tooltip term_id="resource-class" text="ResourceClass">}}
```

Comment:
I think this last point can be removed. The controller doesn't register itself anywhere, it just runs and reacts.

```suggestion
```